dashboard/views/_positions/_ec/_secretary/_position_management.py

```diff
@@ -21,9 +21,11 @@ def secretary_positions(request):
         return HttpResponseRedirect(reverse('dashboard:secretary_positions'))
 
     # doing a .all() ensures that the positions are listed in order of succession since they're created in that order
```

Comment:
Same as above comment. Can remove this comment that is relevant to the removed .all() method